pkg/util/test/fake_snapshot_service.go

```diff
@@ -89,3 +89,31 @@ func (s *FakeSnapshotService) SetVolumeID(pv runtime.Unstructured, volumeID stri
 	s.VolumeIDSet = volumeID
 	return pv, nil
 }
+
```

Comment:
Not sure if this is the right place?

Reply:
Instead of creating a new type, please add `err error` to `FakeSnapshotService` and return `s.err` wherever we need to return an error.